components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/util/I18nEmailUtil.java

```diff
@@ -281,4 +285,44 @@ public static String getNotificationLocale() {
                 ? IdentityUtil.getProperty(I18nMgtConstants.NOTIFICATION_DEFAULT_LOCALE)
                 : I18nMgtConstants.DEFAULT_NOTIFICATION_LOCALE;
     }
+
+    /**
+     * Get the notification template subject, body & footer contents as a byte array.
+     *
+     * @param notificationTemplate  Notification template to get the content
+     * @return                      Byte array of the content
+     */
+    public static byte[] getContentByteArray(NotificationTemplate notificationTemplate) {
+
+        String[] templateContent = new String[]{notificationTemplate.getSubject(), notificationTemplate.getBody(),
+                notificationTemplate.getFooter()};
+        String content = new Gson().toJson(templateContent);
+        return content.getBytes(StandardCharsets.UTF_8);
+    }
+
+    /**
+     * Read the content stream and set the subject, body & footer of the notification template.
+     *
+     * @param contentStream                 InputStream of the content
```

Comment:
Shall we add missing fullstops?

Reply:
Since this is a phrase, not a sentence, its not required to have a full stop.

Reply:
`When writing a phrase, do not capitalize and do not end with a period:`
https://www.oracle.com/lk/technical-resources/articles/java/javadoc-tool.html#@param

Reply:
Infact, phrase should not start with a capital letter, hence fixed that with https://github.com/wso2-extensions/identity-event-handler-notification/compare/8921e51f8b06ac8a0212d3e3920565e66eaf4468..05ec7dcccf7fca80fe209fb72def03d56021562e